tasks/process_agent.py

```diff
@@ -160,6 +160,7 @@ def gen_mocks(ctx):
 
     interfaces = {
         "./pkg/process/procutil": ["Probe"],
+        "./pkg/process/checks": ["Check", "CheckWithRealTime"],
         "./pkg/process/net": ["SysProbeUtil"],
```

Comment:
What do you think about making this alphabetical?

```suggestion
        "./pkg/process/checks": ["Check", "CheckWithRealTime"],
        "./pkg/process/net": ["SysProbeUtil"],
        "./pkg/process/procutil": ["Probe"],
```

Reply:
Yes, this does look better, thank you.